### PR TITLE
Strict ssl

### PR DESCRIPTION
**Pull request by smastous**
Closes jspm/jspm-cli#725

### CODE REVIEW

File: github.js [hunk 1]
```diff
@@ -91,7 +92,8 @@ var GithubLocation = function(options, ui) {
     cwd: options.tmpDir,
     timeout: options.timeout * 1000,
     killSignal: 'SIGKILL',
-    maxBuffer: 2 * 1024 * 1024
+    maxBuffer: 2 * 1024 * 1024,
+    "strict-ssl": this.strictSSL
```
Comment (guybedford) [inline github.js]: This options are specifically for _child_process_ execution options in Node. I don't think `strict-ssl` is an option here?

File: github.js [hunk 2]
```diff
@@ -67,6 +67,7 @@ var GithubLocation = function(options, ui) {
     throw 'Git not installed. You can install git from `http://git-scm.com/downloads`.';
   }
 
+  this.strictSSL = options.strictSSL;
```
Comment (guybedford) [inline github.js]: I think you mean this one to be the `'strictSSL' in config` check?

**Comment by guybedford** [comment]
Thanks so much for pushing this to completion!

File: github.js (hunk 2 above)
Comment (smastous) [inline github.js]: You're right, I missed that one.  Sorry about that.  I have submitted a fix.  Thanks for your patience!

**Pull request merged**